src/Builders/SearchRequestBuilder.php

```diff
@@ -113,6 +117,12 @@ public function sort(string $field, string $direction = 'asc'): self
         return $this;
     }
 
+    public function rescoreRaw(array $rescore): self
```

Comment:
It would be nice to be consistent with other request parameters and introduce also `rescoreQuery` method:

```php
public function rescoreQuery(string $type, array $query): self
{
     $this->rescore['query']['rescore_query'][$type] = $query;
     return $this;
}
```

In addition, we could also include `rescoreWindowSize(int $windowSize)` and `rescoreWeights(int $queryWeight, int int $rescoreQueryWeight)`

Reply:
Good point. I have added them. Can we get elastic-adapter pr merged?

Reply:
Merged, it's part of [v1.16.0](https://github.com/babenkoivan/elastic-adapter/releases/tag/v1.16.0) now. Please, update `elastic-adapter` in `composer.json` and I'll merge this PR.